### ISSUE TITLE
change repo name

### ISSUE THREAD

**Issue by gtallen1187** [telluslabs/kernel-api-python#7]
@thenrich - Now that we're open-sourcing this repo, I'd like to change the name of it to `kernel-python-client`, but didn't want to do so without checking with you first since i'm sure there are plenty of dependencies.  Can you help me change the name to `kernel-python-client` without breaking anything?

**Comment by thenrich** [comment]
Yeah this needs some evaluation first -- I'll try to review shortly.
